spring-batch-test/src/main/java/org/springframework/batch/test/context/SpringBatchTest.java

```diff
@@ -78,7 +81,40 @@
  *
  * }
  * </pre>
+ * 
+ * For JUnit 5, this annotation can be used without having to extend the test class
+ * with {@link SpringExtension} as it is meta-annotated with {@code @ExtendWith(SpringExtension.class)}:
```

Comment:
For the sake of clarity, I would suggest rewording this as follows.

```suggestion
 * For JUnit 5, this annotation can be used without having to manually register the
 * {@link SpringExtension} since {@code @SpringBatchTest} is meta-annotated with
 * {@code @ExtendWith(SpringExtension.class)}:
```